### PR TITLE
document for HTMLAreaElement.target

### PR DESCRIPTION
**Pull request by ShubhamOulkar**
This adds the missing HTMLAreaElement.target page.

Part of the project to document all interoperable features ([mdn/discussions#476](https://github.com/orgs/mdn/discussions/476))

related pr 
https://github.com/mdn/content/pull/31652

https://github.com/mdn/content/pull/31561

### CODE REVIEW

File: files/en-us/web/api/htmlareaelement/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The **`HTMLAreaElement`** interface provides special properties and methods (bey
 
 ## Instance properties
 
-_Inherits properties from its parent, {{domxref("HTMLElement")}}._
+_Inherits properties from its parent {{domxref("HTMLElement")}} and `HTMLHyperlinkElementUtils` mixin._
```
[the remark this answers is not in the document]
Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlareaelement/index.md:16]: Yup

File: files/en-us/web/api/htmlareaelement/target/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+{{ApiRef("HTML DOM")}}
+
+The **`target`** property of the {{domxref("HTMLAreaElement")}} interface is a string that indicates where to display the linked resource.
+
+It reflects the [`target`](/en-US/docs/Web/HTML/Element/area#target) attribute of the {{HTMLElement("area")}} element.
+
+## Value
+
+A string that indicates where to display the linked resource, or the empty string (`""`) if there is no `target` attribute.
+Target values are `_blank`, `_self` (default), `_parent`, `_top`.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlareaelement/target/index.md:18]: We need to do the same as https://github.com/mdn/content/pull/31652/files#r1457060901

(Note there is no default for properties, only for attribute: this property will always return a value)

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/htmlareaelement/target/index.md (hunk 2 above)
Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlareaelement/target/index.md:18]: > (Note there is no default for properties, only for attribute: this property will always return a value)

yup, thank you for suggesting.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**